### PR TITLE
zy/33 add turtle_utils

### PR DESCRIPTION
**Pull request by zheyxu**
For issue RDFLIB: add getFileContent in rdfLib package.
And may need further discussion.

### CODE REVIEW

File: lib/src/turtle_utils.dart
Comment (gjwgit) [inline lib/src/turtle_utils.dart:1]: Perhaps this file should be src/tutrle/parsettl.dart

Will rdflib::parseTTL replace keypod::parseTTLStr?

File: lib/src/graph.dart
Comment (gjwgit) [inline lib/src/graph.dart:1]: Is there an opportunity that this function could go into it's own file?

Comment (zheyxu) [reply lib/src/graph.dart:1]: Hi @gjwgit, need to have more explanations.

**Comment by gjwgit** [comment]
This is a transfer of two functions from keypod - but requires generalisation. Let's close this for now, and other discussion underway.

**Pull request closed without merging**